src/MultiRobot.elm

```diff
@@ -0,0 +1,69 @@
+module MultiRobot exposing (Model, Msg, init, subscriptions, update, view)
+
+import Browser
+import Html
+import Robot
+
+
+main : Program () Model Msg
+main =
+    Browser.element
+        { init = always init
+        , update = update
+        , view = view
+        , subscriptions = subscriptions
+        }
+
+
+type Msg
+    = Robot1Msg Robot.Msg
+    | Robot2Msg Robot.Msg
+
+
+type alias Model =
+    { robot1 : Robot.Model
+    , robot2 : Robot.Model
+    }
+
+
+subscriptions : Model -> Sub Msg
+subscriptions model =
+    Sub.batch
+        [ Sub.map Robot1Msg <| Robot.subscriptions model.robot1
+        , Sub.map Robot2Msg <| Robot.subscriptions model.robot2
+        ]
+
+
+update : Msg -> Model -> ( Model, Cmd Msg )
+update msg model =
+    let
+        movePixels =
+            50
+    in
+    case msg of
+        Robot1Msg robotMsg ->
+            ( { model | robot1 = Robot.update movePixels robotMsg model.robot1 }
+            , Cmd.none
+            )
+
+        Robot2Msg robotMsg ->
+            ( { model | robot2 = Robot.update movePixels robotMsg model.robot2 }
+            , Cmd.none
+            )
```

Comment:
`update : Msg -> Model -> Model` is more suitable, as you always return a `Cmd.none`

Reply:
So what do i need return to instead?

Reply:
So what do I need to return instead?

Reply:
But update and init needs to get the tuple (Model, Cmd Msg)

Reply:
You just add the `Cmd Msg` where it's needed.
For instance:
```elm
init : Model
update : Msg -> Model -> Model
view : Model -> Html Msg
subscriptions : Sub Msg

main : Program () Model Msg
main =
    Browser.element
        { init = always ( init, Cmd.none )
        , update = \msg model -> ( update msg model, Cmd.none )
        , view = view
        , subscriptions = always Sub.none
        }
```